src/app/Main/Body/QueryPage/Results/Content-Results.component.tsx

```diff
@@ -126,9 +132,19 @@ export function ContentResultsComponent(): JSX.Element {
     ]);
 
     return (
-        <ResultsTableResultsComponent
-            displayRowCol={state}
-            columnAttributes={columnAttributes}
+        <QueryPageLoadingOrErrorComponent
+            status={{

```

Comment:
I commented in a previous ticket it would be a good idea to refactor the LoadingErrorComponent so that you directly say if it should be active like LoadingErrorComponent isLoading = true.
So now you see why: setting the status of the api endpoints to 200 here is more like a hack than clean code. Also you have to add for each new api endpoint a new status prop.

Reply:
This will change with the refactoring of the container-view-structure.